conf/openmetadata.yaml

```diff
@@ -383,6 +382,9 @@ email:
   transportationStrategy: ${SMTP_SERVER_STRATEGY:-"SMTP_TLS"}
   templates: ${TEMPLATES:-"openmetadata"}
 
+openMetadataBaseUrlConfiguration:
```

Comment:
@harshach would it make sense to create something like below
global: 
    serverBaseUrl: {}
    other configs: {} 
or we should keep it separate? 
cc @Siddhanttimeline

Reply:
@mohityadav766 @Siddhanttimeline this shouldn't be part of server configs, we should let them configure in system settings and provide openmetadata-ops.sh to configure to help with devops automation